performance_test/src/experiment_configuration/communication_mean.hpp

```diff
@@ -40,6 +41,9 @@ inline std::ostream & operator<<(std::ostream & stream, const CommunicationMean
 {
   if (cm == CommunicationMean::ROS2) {
     return stream << "ROS2";
+  }
+  else if (cm == CommunicationMean::ROS2PollingSub) {
+    return stream <<"ROSPollingSub";
```

Comment:
it should be also ROS2PollingSub

Reply:
@monidzik Sure I have incorporated this change. Thanks for pointing this out.

Reply:
Changed thanks!